docs/source/workflows/performance_workflows.rst

```diff
@@ -0,0 +1,100 @@
+Performance workflows
+=====================
+
+A performance workflow runs cross-validation analysis on a predictor configuration
+
+Registration and validation
```

Comment:
This "Registration and Validation" section is awesome to have in the documentation. I think we should add it to the general AI Engine section since it is applicable to all modules and workflows.

Reply:
sure thing, moved to a more generalized area